airflow/contrib/example_dags/example_gcs_to_bq_operator.py

```diff
@@ -18,46 +18,42 @@
 # under the License.
 
 import airflow
-try:
-    from airflow.contrib.operators import gcs_to_bq
-except ImportError:
```

Comment:
This needs to stay, otherwise the example dags with only a sub-set of extras installs gets nosiy and confusing on the CLI.